packages/common/src/system/nullable-value.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export interface NullableValue<T> {
```

Comment:
why do we need this C# like type in javascript, when js can support null & non null value for any object?

Reply:
Removed